src/tests/ftest/pool/attribute.py

```diff
@@ -198,13 +200,13 @@ def test_pool_attribute_asyn(self):
 
         self.add_pool()
         expected_for_param = []
-        name = self.params.get("name", '/run/attrtests/name_handles/*/')
+        name = self.params.get("name", '/run/attrtests/test_handles/*/')
```

Comment:
This should not be changed - see comment in yaml file.

Reply:
Fixed